app/helpers/hosts_and_hostgroups_helper.rb

```diff
@@ -100,6 +100,60 @@ def realm_field(f, can_override = false, override = false)
             ).html_safe
   end
 
+  def host_puppet_environment_field(f, select_options = {}, html_options = {})
```

Comment:
It's kind of convention - most of methods use it, just scroll up :)
If you still want to change it, I'll do it.